tyk-pro/values.yaml

```diff
@@ -183,13 +230,24 @@ portal:
     #      - chart-example.local
 
 gateway:
+  # Determines whither or not the gateway component should be installed.
   enabled: true
-  kind: DaemonSet
-  replicaCount: 2
+  # The hostname to bind the Gateway to.
   hostName: tyk-gw.local
-  # if enableSharding set to true then you must define a tag to load APIs to these gateways i.e. "ingress"
-  tags: ""
+  # When true, sets the gateway protocol to HTTPS.
   tls: false
+  # Sharding gateway allows you to selectively load APIs to specific gateways. If enabled make sure you have at least one gateway that is not sharded.
+  # Also be sure to match API segmentation tags with the tags selected below.
+  sharding: 
+    enabled: false
+    tag: ""
+  # Set to true to use Tyk as the Ingress gateway to an Istio Service Mesh
+  # We apply some exceptions to the Istio IPTables so inbound calls to the
+  # Gateway and Dashboard are exposed to the outside world - see the deployment templates for implementation
+  enableIstioIngress: false
+
+  kind: DaemonSet
+  replicaCount: 2
```

Comment:
Do we need replicaCount since it's of kind DaemonSet? Please check for others too.

Reply:
Will resolve this in favour of the comment above.